docs/items/weapon-configs.md

```diff
@@ -142,4 +142,450 @@ What values a Weapon Type can have:
+* Trait Set: 26
+  * Weapon Damage: 1.10
+* Config: Combo
+* Ammo: Default
+  * 6+3
+* REQ Teir: 7
+* Game State: 6
+* Notes:-
+
+## [27] Fast Skewer
+* Weapon Type: Skewer + Diminisher Of Hope
+* Trait Set: 27
+  * Weapon Damage: 1.05
+  * Movement Speed: 1.20
+  * Jump Height: 1.30
+  * Reload Speed
+    * Empty Reload Scalar: 3.00
+    * Tactical Reload Scalar: 3.00
+* Config: Combo
+* Ammo: Default
+  * 1+3
+* REQ Teir: 6
+* Game State: 5
+* Notes:-
+
+## [28] M247-Jumper
+* Weapon Type: M247 HMG + Rushdown Hammer
+* Trait Set: 28
+  * Weapon Damage: 1.0
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 4
+* Game State 3
+* Notes: Has a speed and jump boost from Rushdown Hammer
+
+## [29] M247-Striker
+* Weapon Type: M247 HMG + Striker Sidekick
+* Trait Set: 29
+  * Weapon Damage: 0.3
+  * Movement Speed With Turret: 1.20
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 5
+* Game State: 4
+* Notes:-
+  
+## [30] Jorge's Chaingun
+* Weapon Type: M247 HMG + MA40 Longshot
+* Trait Set: 30
+  * Weapon Damage: 1.2
+  * Movement Speed With Turret: 1.47
+  * Damage Resistance
+    * Direct Damage Scalar: 2.10
+    * Grenade Damage Scalar: 1.0
+    * Explosive Damage Scalar: 1.0
+* Config: Scripted
+* Ammo: Default
+  * 100%
+* REQ Teir: 7
+* Game State: 6
+* Notes:-
+  
+## [31] Scrap Cannon (Answer)
+* Weapon Type: Scrap Cannon + Diminisher Of Hope
+* Trait Set: 31
+  * Weapon Damage: 1.7
+  * Movement Speed With Turret: 1.6
+  * Jump Height: 1.13
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 5
+* Game State: 4
+* Notes:-
+
+## [32] Burst Cannon
+* Weapon Type: Plasma Cannon + Rapid Fire Pulse Carbine
+* Trait Set: 32
+  * Weapon Damage: 1.5
+  * Jump Height: 1.13
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 5
+* Game State: 4
+* Notes;-
+
+## [33] Barrage Cannon
+* Weapon Type: Plasma Cannon + Unbound Plasma Pistol
+* Trait Set: 33
+  * Weapon Damage: 1.0
+  * Movement Speed With Turret: 1.4
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 6
+* Game State: 5
+* Notes:-
+
+## [34] Pinpoint Cannon
+* Weapon Type: Plasma Cannon + Pinpoint Needler
+* Trait Set: 34
+  * Weapon Damage: 1.5
+  * Movement Speed With Turret: 1.2
+* Config: Combo
+* Ammo: Default
+  * 100%
+* REQ Teir: 7
+* Game State: 6
+* Notes:-
+
+## [35] adsDamageResistSword
+* Weapon Type: Energy Sword + Unbound Plasma Pistol
+* Trait Set: 35
+  * Weapon Damage: 1.0
+  * Movement Speed: 0.80
+  * Damage Resistance
+    * Direct Damage Scalar: 1.55
+    * Grenade Damage Scalar: 1.0
+    * Explosive Damage Scalar: 1.0
+* Config: Scripted
+* Ammo: Default
+  * 100%
+* REQ Teir: 4
+* Game State: 3
+* Notes:- Movement Speed reduction and Damage Resistance increase, only apply when grounded aiming down sights.
+
+## [36] Prophet's Bane
+* Weapon Type: Infected Energy Sword + MA40 Longshot
+* Trait Set: 36
+  * Weapon Damage: 1.0
+  * VFX - Active Camouflage
+    * Intensity Scalar: 0.80
+    * Interpolation Scalar: 1.00
+* Config: Combo
```

Comment:
Prophet's Bane is scripted cause the camo trait needs to be disabled when there's no ammo

Reply:
Been corrected